externs/browser/w3c_dom1.js

```diff
@@ -312,10 +312,10 @@ Node.TEXT_NODE;
 Node.XPATH_NAMESPACE_NODE;
 
 /**
- * @type {number}
+ * @const {number}
  * @see http://www.w3.org/TR/1998/REC-DOM-Level-1-19981001/level-one-core.html#ID-1950641247
  */
-Node.NOTATION_NODE;
+Node.NOTATION_NODE = 12;
```

Comment:
We generally avoid putting values in the externs, as that keeps them as more clearly a declaration of the "interface" of the external code.

The @const changes look good, though.

Reply:
I hope I can change your mind about that ;)There is a few reasons for supplying explicit values.

Firstly when the `google/elemental2` project reads these externs and sees both a `@const` and a value it is able to explicitly model these values as constant final values in java code. This is much more comfortable experience for elemental2 users to see that a property is constant rather representing it as mutable property in java that generates an error at runtime (GWT2.x) or compile time (j2cl+closure) when you assign a value to it.

Secondly (at least in GWT2.x) it leads to more optimised code. Essentially the compiler no longer needs to use the long symbol (i.e. `Node.NOTATION_NODE`) versus inlining the constant value. I would presume that this is true in closure pipeline as well although I have not verified it.

Thirdly, a lot of sample code or code in the wild actually inlines the constants rather than using constant symbol. (i.e. `WebSocket.readyState` is almost always checked against integers (i.e. `1`) rather than the symbols (i.e. `WebSocket.OPEN`). I am unclear on why this is. Probably people performing a manual optimization ... or copying code that does the same.

The only downsides to including this that I am aware of is the extra effort to add the constants and the possibility that you get something wrong.

Thoughts?